webhooks/start/server.js

```diff
@@ -241,7 +241,7 @@ app.post("/server/receive_webhook", async (req, res, next) => {
 const errorHandler = function (err, req, res, next) {
   console.error(`Your error:`);
   console.error(err);
-  if (err.response?.data != null) {
+  if (err.response != null && err.response.data != null) {
```

Comment:
this change shouldn't be necessary if you're using the recommended version of node, but I'm all for making this project compatible with more node versions, so it seems fine